### ISSUE TITLE
Convert coordinate from ECI to LLA

### ISSUE THREAD

**Issue by Cory-Kramer** [skyfielders/python-skyfield#553]
I've been reading over [the documentation](https://rhodesmill.org/skyfield/api-position.html) and I can't quite figure out how to leverage skyfield to convert a coordinate from earth-centered inertial (ECI, given as x, y, z) to LLA (lat, long, altitude). Consider the following example

```
from datetime import datetime, timedelta
obs_time = datetime(2000, 1, 1) + timedelta(seconds=240)   # observe at 2000-01-01 00:04:00

# Given x, y, z coordinates in ECI [meters]
eci = (2183457.758387, 1924098.065987, 5898755.881731)

# What I would like to do
lat, lon, alt = eci_to_lla(eci, obs_time)

# expected output should be somewhere over Florida
# lat = 63.7398
# lon = 40.3853
# alt = 199473.0 
```

Is there a class in `skyfield.positionlib` that I can use to accomplish this calculation?

**Comment by brandon-rhodes** [comment]
I don't have an editor open on this desktop to try it out, but I'm imagining something like:
```
from skyfield.positionlib import ICRF
p = GCRS([2183457.758387, 1924098.065987, 5898755.881731])
from skyfield.api import wgs84
g = wgs84.subpoint(p)
lat, lon, distance = g.latlon()
```
See whether something like that does what you intend. I'm guessing here that GCRS is a good enough ECI frame for you? You don't mention whether you have a particular one in mind or whether any old ECI frame will do. :)

**Comment by Cory-Kramer** [comment]
Thanks for the code snippet! In that example where does the `GCRS` class come from? I'm getting a `NameError: name 'GCRS' is not defined` and am not getting a hit for that class name in the docs.

**Comment by brandon-rhodes** [comment]
@Cory-Kramer — Now that I'm back at the keyboard, I have made a few edits, and corrected a couple of names; try out the example again and see if it works better this time.

I suppose the documentation needs a new section on coordinate conversion to handle these kinds of cases.

**Comment by Cory-Kramer** [comment]
Thank you @brandon-rhodes that worked perfectly! Just had to tweak the `Distance` line to the following

```
d = [Distance(m=i).au for i in (2183457.758387, 1924098.065987, 5898755.881731)]
```

Thanks again for the help.

**Comment by brandon-rhodes** [comment]
> Thank you @brandon-rhodes that worked perfectly!

Great! Now I just need to figure out where this will go in the documentation. Might take a week or two.

> Just had to tweak the Distance line to the following…

But, if I'm reading correctly, that would leave `d` as a plain Python list. Wouldn't that raise an `AttributeError` in the next line where `d` is asked for its `.au` attribute?

**Comment by Cory-Kramer** [comment]
You're correct, I had to update the line below it as well, edited my comment above to reflect that.
